specs/vendor_spec.rb

```diff
@@ -0,0 +1,5 @@
+require_relative './spec_helper'
+require_relative '../farmar'
+
+require 'simplecov'
```

Comment:
All of these simplecov lines in the specs should only happen once at the beginning of `spec_helper`. Simplecov is pretty picky about that. Please remove the simplecov lines from your individual specs and instead require and start simplecov at the very beginning of `spec_helper`.